src/bin.ts

```diff
@@ -35,6 +35,9 @@ program
             missingKeys.join(', ')
         );
       }
+      if (!options.output && options.channel === 'unlisted') {
+        throw new Error('unlisted addons must have an output');
```

Comment:
How about just using the name from the URL as a fallback?

Reply:
> How about just using the name from the URL as a fallback?

Sure! Could you specify where the URL is stored? Sorry, I'm a bit lost.

Reply:
> > How about just using the name from the URL as a fallback?
> 
> Sure! Could you specify where the URL is stored? Sorry, I'm a bit lost.

I tried to make it that way. I'm hoping it's what you were asking for.